src/components/Modals/Send/SendCommon.tsx

```diff
@@ -9,7 +9,8 @@ export enum SendRoutes {
 export enum SendFormFields {
   Input = 'input', // the raw field input on the address input
   Memo = 'memo', // an optional memo, used only on chains supporting it e.g Cosmos SDK chains
-  Address = 'address', // a valid on chain address
+  To = 'to', // a valid on-chain address
+  From = 'from', // a valid on-chain address
```

Comment:
Since we have `from`, `address` suddenly becomes unergonomic, so renamed to `to` to be consistent